test/modeles/objetsVues/vueAnnexePDFDescription.spec.js

```diff
@@ -49,4 +56,22 @@ describe("L'objet de vue de l'annexe de description", () => {
     expect(donnees).to.have.key('fonctionnalites');
     expect(donnees.fonctionnalites).to.contain('Une fonctionnalité spécifique');
   });
+
+  it('fournit la liste des données stockées', () => {
+    const vueAnnexePDFDescription = new VueAnnexePDFDescription(homologation, referentiel);
+
+    const donnees = vueAnnexePDFDescription.donnees();
+
+    expect(donnees).to.have.key('donneesStockees');
+    expect(donnees.donneesStockees[0]).to.equal('Des données');
+  });
+
+  it('ajoute à la liste des données stockées les spécifiques', () => {
```

Comment:
Je trouverais ça plus simple à lire en répétant « données » :

`it('ajoute à la liste des données stockées les données spécifiques'`

Reply:
J'ai mis `ajoute les données spécifiques à la liste des données stockées` 👍🏼